src/components/Progress/Progress.stories.js

```diff
@@ -11,5 +11,29 @@ const Wrapper = styled.div`
 `;
 storiesOf('Progress', module)
   .addDecorator(story => <Wrapper>{story()}</Wrapper>)
-  .add('default', () => <Progress percent={52} />)
-  .add('no shadow', () => <Progress percent={52} shadow={false} />);
+  .add('default', () => <ProgressExample />)
+  .add('indeterminate', () => <Progress variant='indeterminate' />)
+  .add('hide value', () => <Progress hideValue value={34} />);
+
+const ProgressExample = () => {
+  const [percent, setPercent] = useState(0);
+
+  React.useEffect(() => {
```

Comment:
What do you think about importing `useEffect` the same way you're importing `useState` in order to keep things following the same pattern?

Reply:
fixed in 1d9c842